src/components/appTile/AppTile.js

```diff
@@ -0,0 +1,253 @@
+const GRAVATAR_OPTIONS = "d=identicon&s=60";
+
+const styles = (theme) => ({
+    card: {
+        minWidth: 300,
+        maxWidth: 400,
+        minHeight: 120,
+        maxHeight: 120,
+        margin: 15,
+        padding: 5,
+        cursor: "pointer",
+        "&:hover": {
+            border: "1px solid",
+        },
+    },
+    selectedCard: {
+        border: "1px solid",
+        backgroundColor: palette.lightGray,
+    },
+    avatar: {
+        float: "left",
+        margin: 10,
+    },
+    type: {
+        textAlign: "center",
+        fontSize: 10,
+    },
+    name: {
+        margin: 10,
+        fontSize: 12,
+        textOverflow: "ellipsis",
+        overflow: "hidden",
+        whiteSpace: "nowrap",
+        display: "inline-block",
+        maxWidth: 150,
+        "&:hover": {
+            textDecoration: "underline",
+            cursor: "pointer",
+        },
+    },
+    more: {
+        float: "right",
+        height: 120,
+    },
+    creator: {
+        margin: 10,
+        top: 50,
+        fontSize: 12,
+        fontStyle: "italic",
+        textOverflow: "ellipsis",
+        overflow: "hidden",
+        whiteSpace: "nowrap",
+        display: "inline-block",
+        maxWidth: 150,
+    },
+    status: {
+        position: "relative",
+        top: 40,
+        right: -10,
+    },
+    statusNoMenu: {
+        position: "relative",
+        top: 90,
+        right: 10,
+    },
+    rating: {
+        width: "80%",
+    },
+});
+
+function AppStatus(props) {
+    const { isPublic, isBeta, isDisabled, intl } = props;
+    if (!isPublic) {
+        return (
+            <Tooltip title={formatMessage(intl, "privateAppTooltip")}>
+                <Lock style={{ color: palette.blue }} />
+            </Tooltip>
+        );
+    }
+    if (isBeta) {
+        return (
+            <Tooltip title={formatMessage(intl, "betaAppTooltip")}>
+                <img src={Beta} height={16} alt="beta" />
+            </Tooltip>
+        );
+    }
+    if (isDisabled) {
+        return (
+            <Tooltip title={formatMessage(intl, "disabledAppTooltip")}>
+                <Report style={{ color: palette.red }} />
+            </Tooltip>
+        );
+    }
+    return null;
+}
+
+function AppTile(props) {
+    const {
+        classes,
+        uuid,
+        name,
+        creator,
+        rating,
+        type,
+        isPublic,
+        isBeta,
+        isDisabled,
+        isExternal,
+        onRatingChange,
+        onDeleteRatingClick,
+        MenuItems,
+        intl,
+        onAppNameClicked,
+        onAppSelected,
+        selected,
+    } = props;
+
+    const {
+        average: averageRating,
+        user: userRating,
+        total: totalRating,
+    } = rating;
+
+    const [anchorEl, setAnchorEl] = useState("");
+    const open = Boolean(anchorEl);
+
+    let avatarImgSrc = GRAVATAR_URL + md5(uuid) + "?" + GRAVATAR_OPTIONS;
+
+    const handleAppSelection = () => {};
```

Comment:
Lost this review comment for some reason - is this supposed to return something else?

Reply:
Yes. Good catch!